### PR TITLE
Swat tool 2.4.1

### PR DESCRIPTION
**Pull request by ccondra**
## Purpose of this pull request

This pull request (PR) is to add an overview of the new Site-Wide Analysis Tool (SWAT) to the Reports section of the user guide and a new role to Role Resources. (The main documentation for this tool will be in the knowledgebase.)

## Magento release version

- [ ] 2.4.x

   For the 2.4.1 release

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [x] Magento Commerce only. (Commerce Cloud onlly)

Is this] update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- [Reports](https://docs.magento.com/user-guide/reports.html)
- [Reports Menu](https://docs.magento.com/user-guide/reports/reports-menu.html)
- [Role Resources](https://docs.magento.com/user-guide/system/permissions-role-resources.html)
- [Site-Wide Analysis Tool](https://docs.magento.com/user-guide/reports/site-wide-analysis-tool.html)
- Navigation - operations.yml

## Additional information

Staging: #534

whatsnew
The Site-Wide Analysis Tool (SWAT) provides 24/7 real-time performance monitoring, reports and recommendations to ensure the security and operability of Magento Commerce Cloud installations.

### CODE REVIEW

File: src/reports/reports-menu.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: Reports Menu
 
 Magento provides a wide selection of reports to keep you informed on your marketing efforts, sales products, and customer activity. The Reports menu provides easy access to current information about your sales, products, customers, and promotions.
 
-![]({% link images/images-ee/admin-menu-reports-ee.png %}){: .zoom}
+![]({% link images/images-ee/admin-menu-reports.png %}){: .zoom}
```
Comment (bdenham) [inline src/reports/reports-menu.md:7]:
```suggestion
![Reports Menu]({% link images/images-ee/admin-menu-reports.png %}){: .zoom}
```

Comment (bdenham) [reply src/reports/reports-menu.md:7]: Adding text inside the square brackets (`[Reports Menu]`) gives the image an alt tag for better accessibility.

File: src/reports/site-wide-analysis-tool.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+title: Site-Wide Analysis Tool
+---
+
+The Site-Wide Analysis Tool (SWAT) provides 24/7 real-time performance monitoring, reports and recommendations to ensure the security and operability of Magento Commerce Cloud installations. SWAT provides system insights and recommendations based on best practices that site administrators can use to address detected issues.
```
Comment (bdenham) [inline src/reports/site-wide-analysis-tool.md:5]:
```suggestion
The Site-Wide Analysis Tool (SWAT) provides 24/7 real-time performance monitoring, reports, and recommendations to ensure the security and operability of Magento Commerce Cloud installations. SWAT provides system insights and recommendations based on best practices that site administrators can use to address detected issues.
```

File: src/reports/site-wide-analysis-tool.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+. The service is available to Admin users who have been granted [permission]({% link system/permissions-user-roles.md %}) to access the System Insights > Site-Wide Analysis Reports resource.
+
+![]({% link images/images-ee/reports-swat-dashboard.png %}){: .zoom}
```
Comment (bdenham) [inline src/reports/site-wide-analysis-tool.md:8]:
```suggestion
![Swat Dashboard]({% link images/images-ee/reports-swat-dashboard.png %}){: .zoom}
```

File: src/reports/site-wide-analysis-tool.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+_SWAT Dashboard_
+
+## Access SWAT
+
+On the Admin sidebar, go to ***Reports*** > System Insights > ***Site-Wide-Analysis Tool***.
```
Comment (bdenham) [inline src/reports/site-wide-analysis-tool.md:13]:
```suggestion
On the Admin sidebar, go to **Reports** > System Insights > **Site-Wide-Analysis Tool**.
```

File: src/reports/site-wide-analysis-tool.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+---
+title: Site-Wide Analysis Tool
+---
+
+The Site-Wide Analysis Tool (SWAT) provides 24/7 real-time performance monitoring, reports and recommendations to ensure the security and operability of Magento Commerce Cloud installations. SWAT provides system insights and recommendations based on best practices that site administrators can use to address detected issues.
+. The service is available to Admin users who have been granted [permission]({% link system/permissions-user-roles.md %}) to access the System Insights > Site-Wide Analysis Reports resource.
```
Comment (leslietilling) [inline src/reports/site-wide-analysis-tool.md:6]:
```suggestion
The service is available to Admin users who have been granted [permission]({% link system/permissions-user-roles.md %}) to access the System Insights > Site-Wide Analysis Reports resource.
```

**Comment by leslietilling** [comment]
Please add a `whatsnew` for this PR

**Comment by ccondra** [comment]
running tests

File: src/reports/reports-menu.md [hunk 6]
```diff
@@ -25,6 +25,11 @@ The selection of product [review reports]({% link reports/review-reports.md %})
 
 The selection of [sales reports]({% link reports/sales-reports.md %}) includes Orders, Tax, Invoiced, Shipping, Refunds, Coupons, and settlement reports for PayPal and Braintree.
 
+{:.ee-only}
+### System Insights
+
+The [Site-Wide Analysis Tool]{% link reports/site-wide-analysis-tool.md %}} (SWAT) provides system insights for Commerce Cloud installations of Magento with 24/7 real-time performance monitoring, reports, and recommendations.
```
Comment (cspruiell) [inline src/reports/reports-menu.md:31]: this link doesn't render correctly in staging -- extra } character?

**Pull request merged**